assets/js/modules/thank-with-google/components/settings/SettingsView.js

```diff
@@ -51,30 +81,80 @@ export default function SettingsView() {
 		) || {};
 
 	return (
-		<Fragment>
+		<Grid>
 			<StoreErrorNotices
 				moduleSlug="thank-with-google"
 				storeName={ MODULES_THANK_WITH_GOOGLE }
 			/>
 
-			<div className="googlesitekit-settings-module__meta-items">
-				<div className="googlesitekit-settings-module__meta-item">
+			<Row>
+				<Cell className="googlesitekit-settings-module__meta-item">
```

Comment:
@eugene-manuilov I understand your point on being consistent. However, all our existing settings pages use flexbox for each row because the number of settings are inconsistent on every row. The  `googlesitekit-settings-module__meta-items` class solely creates this flexbox, which we actually don't need here. TwG is the only settings layout which consistently has two settings per row equally spaced forming a proper grid. I guess that is why the IB suggested to use `<Grid>`, `<Row>` and `<Cell>`?

Reply:
Ok, let it be it. We will address it in the future if a need be.